### DISCUSSION TITLE
Negative labels in MAG240

### DISCUSSION THREAD

**Discussion by sandeep06011991** [snap-stanford/ogb#442] [MAG240M-LSC]
Some of the labels of the training vertices in MAG240M are < 0. 
This will break the entropy loss calculation.
Can I get some clarification on why the labels are negative.

**Comment by rusty1s** [comment]
Training labels should be < 0 for nodes for which we don't have ground-truth information. These labels are not part of `split_idx['train']` and should not be used for supervision. However, the nodes, connections and features can be used within the GNN.
